pkg/domain/entities/images.go

```diff
@@ -244,6 +244,7 @@ type ImagePushOptions struct {
 	// integers in the slice represent 0-indexed layer indices, with support for negative
 	// indexing. i.e. 0 is the first layer, -1 is the last (top-most) layer.
 	OciEncryptLayers *[]int
+	AddCompression   []string
```

Comment:
Do we need to worry about `podman push` (i.e. not a `manifest push`) with `AddCompression` set?

(That’s the price of sharing the same `ImagePushOptions` for two operations…)

I can see the argument that the only reasonable callers are from inside the Podman codebase, so we don’t care…

Still, probably worth at least adding a comment that this only applies to manifest push, _for now_. (I think we will want to add that in the future.)